src/bin/add/main.rs

```diff
@@ -126,7 +126,12 @@ fn is_sorted(mut it: impl Iterator<Item = impl PartialOrd>) -> bool {
 }
 
 fn unrecognized_features_message(message: &str) -> Result<()> {
-    let bufwtr = BufferWriter::stderr(ColorChoice::Always);
+    let colorchoice = if concolor_control::get(concolor_control::Stream::Stderr).color() {
+        ColorChoice::Always
+    } else {
+        ColorChoice::Never
+    };
```

Comment:
This should ideally be a helper function, so you can just do `BufferWriter::stderr(crate::colorchoice());` below. Maybe even cached.

Reply:
concolor_control handles caching for us.

What, if any, opinions do we have on the public API for `cargo-edit`?   Before, I've always ignored the use case of people depending on `cargo-edit` (wish there was a good way to publish a `[[bin]]` and make clear the `[lib]` is purely for organizational purposes) but #557 made me go down this rabbit hole and now wondering about how much to worry about what we put into it.

Making this a function includes
- Exposing `termcolor` in the public API
- Putting this in the top-level of the API unless we create a `pub mod` just for this

We'd also want to make sure we include `stderr` in the function name so people don't misuse it (granted, we were already mixing stdout and stderr).

> I assume "concolor-control" is for "console color control"? that's a tough name to read and write! I'm tempted to suggest importing it as "cccolor

"concolse color control" and spanish for "with color".  I started off being explicit with each of the crates but seeing how they developed, `concolor-control` is the primary crate, so it could make sense to rename it to `concolor`.

Reply:
> We really only need to expose ColorChoice, right?

Correct, only exposing that.  It still couples our public API to theirs (upgrading to a breaking change is a breaking change, changing implementations is a breaking change).

With your answers, I'll go ahead and make the change.